src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerIntellisenseWorkspace.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.DebuggerIntelliSense
```

Comment:
@CyrusNajmabadi That'd cause a lot of other downstream problems (some of this is a public API after all.) Am I missing something here?